pay-api/src/pay_api/resources/fas/routing_slip_links.py

```diff
@@ -0,0 +1,57 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Resource for Account payments endpoints."""
+from http import HTTPStatus
+
+from flask import current_app, jsonify
+from flask_restx import Namespace, Resource, cors
+
+from pay_api.services.fas import RoutingSlipService
+from pay_api.utils.auth import jwt as _jwt
+from pay_api.utils.enums import Role
+from pay_api.utils.trace import tracing as _tracing
+from pay_api.utils.util import cors_preflight
+
+
+API = Namespace('fas', description='Fee Accounting System Links')
+
+
+@cors_preflight('GET,PATCH')
+@API.route('/', methods=['GET', 'PUT', 'OPTIONS'])
+class RoutingSlip(Resource):
+    """Endpoint resource update and return routing slip by number."""
```

Comment:
change description and may be rename the class as RoutingSlipLinks would make more sense.